account_cash_discount_base/README.rst

```diff
@@ -0,0 +1,2 @@
+Account Cash Discount Base
+===
```

Comment:
underlines the entire sentence
A somewhat more complete description would be nice. :smile: